backend/globaleaks/handlers/base.py

```diff
@@ -225,6 +227,24 @@ def call_handler(cls, *args, **kwargs):
 
         return wrapper
 
+    def basic_auth(self):
```

Comment:
I have seen decorators in this code which make a response take a default amount of time to return. Can this be applied here?